packages/theme-patternfly-org/app.js

```diff
@@ -7,39 +7,41 @@ import { MDXTemplate } from 'theme-patternfly-org/templates/mdx';
 import { routes, groupedRoutes, fullscreenRoutes, getAsyncComponent } from './routes';
 import 'client-styles';
 
-const AppRoute = ({ child }) => {
+const AppRoute = ({ child, katacodaLayout }) => {
   const location = useLocation();
   if (typeof window !== 'undefined' && window.ga) {
     window.ga('set', 'page', location.pathname);
     window.ga('send', 'pageview');
   }
-
   return (
     <React.Fragment>
       {child}
-      {process.env.hasFooter && <Footer />}
+      {!katacodaLayout && process.env.hasFooter && <Footer />}
     </React.Fragment>
   );
 }
 
-const SideNavRouter = () => (
-  <SideNavLayout groupedRoutes={groupedRoutes}>
-    <Router id="ws-page-content-router">
-      {Object.entries(routes)
-        .map(([path, { Component, title, sources }]) => Component
-          ? <AppRoute key={path} path={path} default={path === '/404'} child={<Component />} />
-          : <AppRoute key={path} path={path + '/*'} child={
-              <MDXTemplate
-                path={path}
-                title={title}
-                sources={sources}
-              />
-            }/>
-        )
-      }
-    </Router>
-  </SideNavLayout>
-);
+const SideNavRouter = () => {
+  const pathName = useLocation().pathname;
```

Comment:
I'm not in favor of changing the layout based on path name. Can you implement this using `useLocation().state` instead?

```js
const isNavOpen = !useLocation().state.isKatacoda;
...
<SideNavLayout isNavOpen={isNavOpen}
```